components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementAdminService.java

```diff
@@ -327,6 +327,22 @@ public void removeWorkflow(String id) throws WorkflowException {
         }
     }
 
+    /**
+     * Remove workflows by a given tenant id.
+     *
+     * @param tenantId Id of workflow to remove
+     * @throws WorkflowException
```

Comment:
Fix the comments properly. Check all the places.

Reply:
Fixed in #3312